vnext/ReactUWP/Views/TouchEventHandler.cpp

```diff
@@ -427,6 +446,42 @@ bool TouchEventHandler::TagFromOriginalSource(
   return true;
 }
 
+winrt::IPropertyValue TouchEventHandler::TestHit(
+    const winrt::Collections::IVectorView<xaml::Documents::Inline> &inlines,
+    const winrt::Point &pointerPos) {
+  winrt::IPropertyValue tag(nullptr);
+  for (const auto &el : inlines) {
+    if (const auto span = el.try_as<winrt::Documents::Span>()) {
+      const auto resTag = TestHit(span.Inlines().GetView(), pointerPos);
+
+      if (resTag) {
+        return resTag;
+      }
+
+    } else if (const auto run = el.try_as<winrt::Documents::Run>()) {
+      const auto start = el.ContentStart();
+      const auto end = el.ContentEnd();
+
+      const auto startRect = start.GetCharacterRect(winrt::Documents::LogicalDirection::Forward);
+      const auto endRect = end.GetCharacterRect(winrt::Documents::LogicalDirection::Forward);
+
+      // Approximate the bounding rect (for now, don't account for text wrapping or right-to-left orientation)
+      if ((startRect.X <= pointerPos.X) && (endRect.X + endRect.Width >= pointerPos.X) &&
+          (startRect.Y <= pointerPos.Y) && (endRect.Y + endRect.Height >= pointerPos.Y)) {
+        tag = el.GetValue(winrt::FrameworkElement::TagProperty()).try_as<winrt::IPropertyValue>();
+        if (tag) {
+          return tag;
+        } else {
+          // TODO what if we land here? Can there be any more elements that satisfy the pointer position test?
+          // Why was the tag property not set?
+        }
```

Comment:
This seems like it could only happen if there was a bug.  You could do an "assert(tag)" instead of "if (tag)"